coreblocks/params/configurations.py

```diff
@@ -74,7 +74,15 @@ def replace(self, **kwargs):
 full_core_config = CoreConfiguration(
     isa_str="rv32imzicsr",
     func_units_config=(
-        RSBlockComponent([ALUComponent(), JumpComponent()], rs_entries=4),
+        RSBlockComponent(
+            [
+                # Separate ALU for shifting
+                AluBuilder(tag_set=InstTag.ANY ^ InstTag.SHIFT).build_component(),
```

Comment:
idk if readable enough
```suggestion
                AluBuilder(tag_set=InstTag.ANY & ~InstTag.SHIFT).build_component(),
```